### PR TITLE
Add documentation for workspaces CLI.

### PR DESCRIPTION
**Pull request by afshin**
This PR updates the user documentation re: workspaces command-line interface.

### CODE REVIEW

File: docs/source/user/urls.rst [hunk 1]
```diff
@@ -25,23 +25,22 @@ Entering this URL will open the notebook in JupyterLab in
...
+Managing Workspaces (UI)
+~~~~~~~~~~~~~~~~~~~~~~~~
 
 JupyterLab sessions always reside in a workspace. Workspaces contain the state
 of JupyterLab: the files that are currently open, the layout of the application
 areas and tabs, etc. When the page is refreshed, the workspace is restored.
 
-The default workspace is not named and only saves its state on the
-user's local browser:
+The default workspace is not named, it is the primary ``/lab`` URL.
```
Comment (ivanov) [inline docs/source/user/urls.rst]: "it resides at the primary ``/lab`` URL." ? or "is accessed via the primary ``/lab`` URL."? Also would it be better to start with "The default workspace does not have a name"? "...is not named" sounds like maybe you could name it, but it just isn't named by default.

Comment (afshin) [reply docs/source/user/urls.rst]: Cheers, @ivanov, will update it in the morning. Hopefully CI will be happy then, too.

**Comment by jasongrout** [comment]
@ivanov, want to take a look at this now?

File: packages/filebrowser-extension/package.json [hunk 2]
```diff
@@ -34,7 +34,6 @@
     "@jupyterlab/apputils": "^0.19.1-alpha.0",
     "@jupyterlab/coreutils": "^2.2.1-alpha.0",
     "@jupyterlab/docmanager": "^0.19.1-alpha.0",
-    "@jupyterlab/docregistry": "^0.19.1-alpha.0",
```
Comment (ivanov) [inline packages/filebrowser-extension/package.json]: did this sneak in accidentally?

File: tests/test-cells/src/widget.spec.ts [hunk 3]
```diff
@@ -509,14 +509,31 @@ describe('cells/widget', () => {
...
+        // Count how many update requests were processed.
+        await framePromise();
+        const original = widget.methods.filter(method => {
+          return method === 'onUpdateRequest';
+        }).length;
+
         widget.rendered = false;
         widget.rendered = false;
+        widget.rendered = false;
+        widget.rendered = false;
```
Comment (ivanov) [inline tests/test-cells/src/widget.spec.ts]: these look like unrelated changes.... did this also sneak in accidentally?

File: packages/filebrowser-extension/tsconfig.json [hunk 4]
```diff
@@ -18,9 +18,6 @@
     {
       "path": "../docmanager"
     },
-    {
-      "path": "../docregistry"
-    },
```
Comment (ivanov) [inline packages/filebrowser-extension/tsconfig.json]: did this sneak in accidentally?

File: packages/filebrowser-extension/package.json (hunk 2 above)
Comment (jasongrout) [reply packages/filebrowser-extension/package.json]: I fixed this too in https://github.com/jupyterlab/jupyterlab/pull/5436. You're right it's unrelated.

File: packages/filebrowser-extension/tsconfig.json (hunk 4 above)
Comment (jasongrout) [reply packages/filebrowser-extension/tsconfig.json]: This is part of the dependency fix above.

**Comment by jasongrout** [comment]
You're right there are some unrelated changes in here to fix some failing tests or integrity updates. Looks good to me, though I think we should also iterate more on that markdown cell test.

**Comment by afshin** [comment]
Cheers!

@ivanov I didn't want to cross-pollinate but after a rebase there was a test failure and a dependency warning in my branch. Thanks for reviewing!

**Comment by jasongrout** [comment]
FYI, I'm still sporadically getting that markdown test failure, so the fix is still out there somewhere.
